docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2718,6 +2807,17 @@ parameters:
     required: false
     default: 0
 
+  MaxUncommittedEvents:
+    name: max_uncommitted_events
+    in: query
+    description: |
+      The amount of uncommitted events Nakadi will stream before pausing the stream. When in paused
```

Comment:
the maximum amount

Reply:
I thought that this is the actual number that will be sent before pause. Is there the usecase when Nakadi will pause before `max_uncommitted_events` is reached?

Reply:
What if there are no more events to send?

Reply:
Ok, you win :)
https://github.com/zalando/nakadi/pull/834/commits/ea4706bee6c92e524ea4f7b3cd71df03ceea9975